src/components/pages/photos/elements/aboutText/aboutText.js

```diff
@@ -0,0 +1,21 @@
+import roverNames from '../../../../../enums/roverNames';
+import cameraNames from '../../../../../enums/cameraNames';
+
+export default class AboutText {
+  constructor() {
+    Object.values(roverNames).forEach(rover => { this[rover] = `${rover} yay`; });
+    Object.values(cameraNames).forEach(camera => { this[camera] = `${camera} yay`; });
```

Comment:
It looks like we've got text for all of these now, so we can get rid of these testing lines!